kkarsnia/BarFinder-ATX
# Updating search to show results nearest to user.

And reflect the true distance of a venue to the user.

**Why are We Doing This?**
Having the distance be calculated from the center of town doesn't do the user much good when they are looking for something close to them. It also makes it easier to show the closest bars to the user. 

**Steps to Test**

- Run the app

- Click the button that says Cool, Show Bars Near Me!

- Observe the bars list sorted by closest to the user.

- Observe the updated distance to the user to the right of each venue name/category
